client/js/widgets/widget.js

```diff
@@ -2067,4 +2071,11 @@ export class Widget extends StateManaged {
   validDropTargets() {
     return getValidDropTargets(this);
   }
+
+  zArray() {
+    if(this.get('parent') && widgets.has(this.get('parent')))
+      return widgets.get(this.get('parent')).zArray().concat(this.z)
```

Comment:
If not using the computed css property, I think this might need to account for the widget's layer as well, similar to calculateZ:
https://github.com/ArnoldSmith86/virtualtabletop/blob/b4d57d21e924d3b73db20a4bb2b2f518d0c67e0a/client/js/widgets/widget.js#L253

Reply:
Thanks for clarifying, sorry for the noise! I'm new to the code 😇